hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/TxnRecordRegression.java

```diff
@@ -108,16 +108,17 @@ private HapiSpec recordNotFoundIfNotInPayerState() {
                         cryptoCreate("misc").via("success"),
                         usableTxnIdNamed("rightAccountWrongId").payerId("misc"))
                 .when()
-                .then(getTxnRecord("rightAccountWrongId").hasCostAnswerPrecheck(RECORD_NOT_FOUND));
+                .then(getTxnRecord("rightAccountWrongId").hasAnswerOnlyPrecheck(RECORD_NOT_FOUND));
```

Comment:
@Neeharika-Sompalli @tinker-michaelj is this change correct?

Reply:
I am wondering this too. We normally should not change the test itself as it also needs to run in mono code successfully.

Reply:
Not sure too. Why do we need this change @MrValioBg  ?